src/hotspot/share/memory/allocation.hpp

```diff
@@ -146,6 +146,7 @@ class AllocatedObj {
   f(mtServiceability, "Serviceability")                                              \
   f(mtMetaspace,      "Metaspace")                                                   \
   f(mtStringDedup,    "String Deduplication")                                        \
+  f(mtObjectMonitor,  "Monitors")                                                    \
```

Comment:
I would use "ObjectMonitors" in the string to distinguish from VM internal Monitors.
It also matches the `mtObjectMonitor` name better.